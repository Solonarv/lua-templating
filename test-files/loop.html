<%
-- check that interleaving text/lua mode switches
-- and lua blocks works properly, and also loops
function do_item(i) %> <li><%=i%>th item</li> <% end %>
<ul>
<% for i = 1,10 do do_item(i) end %>
</ul>
