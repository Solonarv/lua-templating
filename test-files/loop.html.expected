
<ul>
 <li>1th item</li>  <li>2th item</li>  <li>3th item</li>  <li>4th item</li>  <li>5th item</li>  <li>6th item</li>  <li>7th item</li>  <li>8th item</li>  <li>9th item</li>  <li>10th item</li> 
</ul>
